common/src/chain/pos.rs

```diff
@@ -106,14 +106,17 @@ impl PoSChainConfig {
 }
 
 pub fn create_testnet_pos_config() -> PoSChainConfig {
+    let target_block_time = NonZeroU64::new(2 * 60).expect("cannot be 0");
+    let target_limit = Uint256::MAX / Uint256::from_u64(target_block_time.get());
+
     PoSChainConfig {
-        target_limit: Uint256::MAX,
-        target_block_time: NonZeroU64::new(2 * 60).expect("cannot be 0"),
+        target_limit,
+        target_block_time,
         reward_maturity_distance: 2000.into(),
         decommission_maturity_distance: 2000.into(),
         spend_share_maturity_distance: 2000.into(),
-        block_count_to_average_for_blocktime: 5,
-        difficulty_change_limit: PerThousand::new(100).expect("must be valid"),
+        block_count_to_average_for_blocktime: 100,
+        difficulty_change_limit: PerThousand::new(1).expect("must be valid"),
```

Comment:
1/1000 change limit seems pretty tight. Is the adjustment done on every block?

Reply:
Yes

Reply:
right, that's tight because we adjust per block